apps/doji-backend/src/assets/reset-password.hbs

```diff
@@ -0,0 +1,12 @@
+<h1>
+  A request has been received to change the password for your Doji account.
+</h1>
+<p>
+  If you did not request this change, please ignore this email.
+</p>
+<p>
+  To reset your password, please click the link below.
+</p>
+<p>
+  <a href='https://{{domain}}/auth/reset-password/{{token}}'>Reset Password</a>
```

Comment:
I got

`https://https//dev.choco.saenyakorn.dev/auth/reset-password/ca...`

In my email.

Reply:
OK

Reply:
ที่ตั้งใจไว้คือ จะให้ backend ส่ง link frontend/auth/reset-password/:token ไป โดย token อายุ 5 min

เมื่อกดที่ link ที่ได้ใน mail จะให้ไปหน้าเว็บแล้วกรอก password / confirmPassword แล้ว POST ไปที่ backend/auth/reset-password/:token (หรือมันควรอยู่ใน body หว่า)

จากนั้นค่อยไป update record ใน db